docs/api/services.md

```diff
@@ -0,0 +1,308 @@
+---
+id: services
+title: 🥡 Services
+sidebar_position: 9
+---
+
+Services provide external services needed to run the application while developing it. They are supposed to be used while developing and testing the application.
+
+## Service Interface
+
+`Service` is an interface that defines the methods for a service.
+
+### Definition
+
+```go
+// Service is an interface that defines the methods for a service.
+type Service interface {
+    // Start starts the service, returning an error if it fails.
+    Start(ctx context.Context) error
+
+    // String returns a string representation of the service.
+    // It is used to print the service in the startup message.
+    String() string
+
+    // State returns the current state of the service.
+    State(ctx context.Context) (string, error)
+
+    // Terminate terminates the service, returning an error if it fails.
+    Terminate(ctx context.Context) error
+}
+```
+
+## Methods on the Service
+
+### Start
+
+Start starts the service, returning an error if it fails. This method is automatically called when the application starts.
```

Comment:
The first "Start" is redundant

Reply:
Oh my! copied from the go docs. Removing it asap